spec/controllers/state_file/questions/nj_household_rent_own_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "rails_helper"
+
+RSpec.describe StateFile::Questions::NjHouseholdRentOwnController do
+  let(:intake) { create :state_file_nj_intake }
+  before do
+    sign_in intake
```

Comment:
I think so... I was actually borrowing from other controller test code... maybe a CFA eng can answer this for sure?

Reply:
I dont think sign_in intake instantiates the controller but you will need it though to simulate a user being signed in because some of your tests might redirect to the login page if the test does not find an authenticated user.